### DISCUSSION TITLE
Penggunaan Reactive di Shiny untuk Multiple Filter

### DISCUSSION THREAD

**Discussion by ghost** [teamalgoritma/community#708] [Q&A] [answered]
Bagaimana cara penggunaan reactive dalam Shiny. Saya ingin melakukan filter dalam beberapa tahapan, namun ternyata setelah mencari tahu tidak bisa menggunakan filter manual berkali-kali. 

![image](https://github.com/teamalgoritma/community/assets/126555458/cd3a6c51-8695-49fa-a340-f5da4f85582d)
hasilnya terdapat error 

## server.R
output$box_filter_criteria <- renderValueBox({
      
      filter_reactive  <- paste0(nrow(fire_reactive()), ' (', round(nrow(fire_reactive()) * 100 / nrow(fire_clean), 1), '%)')
      
     valueBox("# Fire Incidents Matching Filter Criteria", value = filter_reactive, icon="filter", color = "orange")
    })
    

## ui.R
valueBoxOutput('box_filter_criteria',
                                    width = 3)

**Comment by idoali** [comment, marked as answer]
Dear Pak Akmal

Seperti yang sudah kita bicarakan di hari Jum'at kemarin ya pak, kita bisa menyelesaikan masalah ini dengan fungsi `textOutput()` pada file ui.R. Untuk detailnya bisa bapak lihat di halaman ini ya pak https://shiny.posit.co/r/reference/shiny/1.7.4/textoutput 

Terima kasih
